teachopencadd/talktorials/018_md_simulation/talktorial.ipynb

```diff
@@ -0,0 +1,1155 @@
+{
```

Comment:
<p>I saw in some other notebooks that they adapted the figure size, maybe one could apply this here too? check <a href="https://github.com/volkamerlab/teachopencadd/blob/tbk-016-revamp/teachopencadd/talktorials/T016_protein_ligand_interactions/talktorial.ipynb" target="_blank">T016</a> figure 1</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>

Reply:
<p>I thought I remembered from our first Hackathon that we want to avoid HTML Syntax, which is the only way I know to scale images in notebooks. But I cannot find the guidelines anymore and might be mistaken. Maybe Jaime can tell us.</p>